.github/workflows/generate_sdk.yaml

```diff
@@ -16,7 +13,7 @@ jobs:
       - uses: craicoverflow/openapi-generator-action@v1
         with:
           generator: go
-          input: ${{github.event.inputs.openapiSpecUrl}}
+          input: ${{ github.event.client_payload.openapi_file_path }}
```

Comment:
I am assuming you want to access the below information you sent as payload in the repository dispatch
```
client-payload: '{"openapi_file_path": "app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json"}'
```